packages/dds/tree/src/core/fence.json

```diff
@@ -0,0 +1,19 @@
+{
+  "tags": ["core"],
+  "exports": ["index"],
+  // Depending on "shared-tree-core" so implementations of Indexes can be here.
+  // Could move the index abstraction elsewhere instead.
+  "imports": [
```

Comment:
I'm curious why `edit-manager` is not in there. Are you not considering part of the core?

Reply:
It just happens to have no exports that are used by users of core right now, so I missed including it here. I have now added it. Looks like it was the only one missing.

I think this makes a good argument for placing all these libraries in a "core" directory, so we don't rely on this list or the readme to clarify where things live. If we do want to do that, I'd recommend a separate PR for it since I dislike mixing changes with moves.